src/resources/views/editor.blade.php

```diff
@@ -1,6 +1,5 @@
-(function(window,$){
```

Comment:
This file can be reverted in favor of https://github.com/yajra/laravel-datatables-html/pull/139